### PR TITLE
Format "Downloaded" message

### PR DESCRIPTION
**Pull request by lukeed**
Following the conversation from https://github.com/vutran/zel/pull/6#issuecomment-287600218

Output now looks like:

screen shot 2017-03-19 at 2 03 28 am

> Printing dummy files because I hit the Rate Limit 😅 

Also updates ESLint rule for `arrow-parens`.

### CODE REVIEW

File: lib/constants.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
     DOTFILE: '.zel',
     ERROR: chalk.red('Error:'),
     OK: chalk.green('Downloaded:'),
-    FROM: chalk.gray('from'),
+    REPO: chalk.cyan.underline,
```
Comment (vutran) [inline lib/constants.js]: May be better to rename to `DIVIDER` for other purposes later if needed

Comment (lukeed) [inline lib/constants.js]: Rename `REPO` or `SPACER`?

Comment (vutran) [inline lib/constants.js]: `REPO`

**Comment by lukeed** [comment]
Even cooler would be a `zel login`, with authorizes the user with GitHub & grabs a token. Then we store their token locally somewhere (`~/.zel` like yarn) and use that for all their API calls.

It bumps them from 60/hr to 5000/hr. Might be necessary for deeply-chained projects.

Of course, that dir would then be the place to cache repo files too.

File: lib/constants.js (hunk 1 above)
Comment (lukeed) [inline lib/constants.js]: How about to `TITLE`?

**Comment by vutran** [comment]
> Even cooler would be a zel login, with authorizes the user with GitHub & grabs a token. Then we store their token locally somewhere (~/.zel like yarn) and use that for all their API calls.

I was thinking about this earlier but may pose a risk in that other apps may try to read from the user file (`~/.zel`) and grab the token for their own usage.

File: lib/constants.js (hunk 1 above)
Comment (vutran) [inline lib/constants.js]: SGMT

**Comment by lukeed** [comment]
There oughta be a way to way to encrypt it or lock the file...

File: index.js [hunk 2]
```diff
@@ -4,24 +4,28 @@ const prog = require('caporal');
...
+const { ERROR, OK, REPO, SPACER } = require('./lib/constants');
 
 function init(repo, logger) {
-    downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
-    }).catch(err => logger.error(ERROR, err));
+    downloadRepo(repo)
+        .then((arr) => arr.map((file) => `${SPACER} - ${file}`))
+        .then((arr) => [REPO(repo)].concat(arr, ''))
+        .then((arr) => logger.info(OK, arr.join('\n')))
```
Comment (vutran) [inline index.js]: Looking at this is a bit confusing at first.

A simpler approach would just be to log the title separately from the files.

```js
logger.info('Downloaded: vutran/editorconfig'); // formatted with colors, spacers, etc.
logger.info(arr); // I believe Caporal should reformat arrays of literals as <ul>, can `map` before if needed
```

Comment (lukeed) [inline index.js]: Yeah. I just wanted to do it in single lines. 😼

**Comment by lukeed** [comment]
@vutran Do we still want this kind of success output?

**Comment by vutran** [comment]
I'm thinking that we can separate status outputs and a final output. Status would simply display what is currently being downloaded along with the repo name (see your previous comment in #6 ) and final could just be a tree structure. This mimics the `npm` client's behavior a little bit.

**Pull request merged**